gammapy/irf/irf_stack.py

```diff
@@ -0,0 +1,94 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+import numpy as np
+from astropy.units import Quantity
+from ..irf import EffectiveAreaTable, EnergyDispersion
+
+__all__ = [
+    'IRFstack',
+]
+
+log = logging.getLogger(__name__)
+
+
+class IRFstack(object):
+    """
+    Compute the mean effective area and the mean rmf for several observations for a given list of rmf and arf
+
+    Parameters
+    ----------
+    list_arf: list
+        list of arf
+    list_livetime: list
+        list of livetime
+    list_rmf: list
+        list of rmf
+    list_low_threshold: list
+        list of low energy threshold (for the rmf mean computation)
+    list_high_threshold: list
+        list of high energy threshold (for the rmf mean computation)
+
+
+    """
+
+    def __init__(self, list_arf, list_livetime, list_rmf=None, list_low_threshold=None, list_high_threshold=None):
+        self.list_arf = list_arf
+        self.list_rmf = list_rmf
+        self.list_livetime = list_livetime
+        self.stacked_aeff = None
+        self.stacked_edisp = None
+        self.list_low_threshold = list_low_threshold
+        self.list_high_threshold = list_high_threshold
+
+    def mean_arf(self):
+        """
+        Compute the mean arf for several observations weighted by the livetime.
+
+        Create a new `~gammapy.irf.EffectiveAreaTable` with the mean arf
```

Comment:
I put the formulas for the stacking here
http://docs.gammapy.org/en/latest/api/gammapy.spectrum.SpectrumObservationStacker.html#gammapy.spectrum.SpectrumObservationStacker

Can you copy them over here?